docs/api/apiv3/paths/custom_field_item_branch.yml

```diff
@@ -0,0 +1,61 @@
+# /api/v3/custom_field_items/{id}/branch
+---
+get:
+  summary: Get a custom field hierarchy item's branch
+  operationId: get_custom_field_item_branch
+  description: |-
+    Retrieves the branch of a single custom field item specified by its unique identifier.
```

Comment:
🟡 I'd like to explain here, what we understand under the term `branch`. Something like.

> A branch is list of all ancestors, starting with the root item and finishing with the item itself.

Reply:
Interesting. Intuitively I would have assumed that a branch is everything below an item (descendants) and not above (ascendants). So, apparently it is not clear to me not to @Kharonus.

Ideas:
- ItemWithAscendants
- ItemWithPath (inspired by Graph API)